### ISSUE TITLE
Update table failed - renaming column and changing field length

### ISSUE THREAD

**Issue by jrvidotti** [nocodb/nocodb#1018]
Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mssql**
ProjectOnRootDB: **false**
RootDB: **mssql**
PackageVersion: **0.84.8**

**Steps To Reproduce**
When renaming a column and changing field length in the same operation, I get an error "Update table failed". In the database, the column is renamed but the length isn't changed. Inside the application, the name stills the old one and I can't operate the field anymore.

If I rename the column and after that I change the length, it works fine!

**Comment by wingkwong** [comment]
It seems the the second statement fails because it couldn't find the column name as it has been renamed. 

Changes:
```
title -> title2
varchar(45) -> varchar(46)
```

Log: 
```
EXEC sp_rename 'dbo.table_a.title', 'title2', 'COLUMN';

ALTER TABLE [dbo].[table_a] ALTER COLUMN [title] varchar(46);
 - ALTER TABLE ALTER COLUMN failed because column 'title' does not exist in table 'table_a'.
```

**Comment by wingkwong** [comment]
Fixed in ``0.84.9``.
